drf_braces/tests/test_utils.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import print_function, unicode_literals
+
+import unittest
+
+from drf_braces.utils import (
+    get_class_name_with_new_suffix,
+    get_attr_from_base_classes,
+)
+from rest_framework import fields
+
+
+class TestUtils(unittest.TestCase):
+    def test_get_class_name_with_new_suffix(self):
+        new_name = get_class_name_with_new_suffix(
+            klass=fields.IntegerField,
+            existing_suffix='Field',
+            new_suffix='StrawberryFields'
+        )
+        self.assertEqual(new_name, 'IntegerStrawberryFields')
+
+        new_name = get_class_name_with_new_suffix(
+            klass=fields.IntegerField,
+            existing_suffix='straws',
+            new_suffix='Blueberries'
+        )
+        self.assertEqual(new_name, 'IntegerFieldBlueberries')
+
+    def test_get_attr_from_base_classes(self):
+        Parent = type(str('Parent'), (), {'fields': 'pancakes'})
+
+        self.assertEqual(
+            get_attr_from_base_classes((Parent,), [], 'fields'), 'pancakes'
+        )
+
+        self.assertEqual(
+            get_attr_from_base_classes(
+                (Parent,), {'fields': 'mushrooms'}, 'fields'
+            ),
+            'mushrooms'
+        )
+
+        self.assertEqual(
+            get_attr_from_base_classes((Parent,), [], '', default='maple_syrup'),
```

Comment:
not following.

`get_attr_from_base_classes` pretty does what Python does internally when class is already created:

when looking for attribute `foo`

```
# stupid implemention which only works for classes
def getattr(obj, attr, default=None):
    for i in attr.mro():  # mro includes itself
        if attr in i.__dict__:
            return i.__dict__['attr']
    if default is not None:
        return default
    raise AttributeError
```

except in my case I dont have access to mro since class has not been created yet....